pinc/theme.inc

```diff
@@ -270,43 +369,52 @@ function html_navbar()
         // on the same line.
         echo "<span id='navbar-right'>";
         $links = [];
-        if (get_url_for_inbox()) {
-            $inbox_text = _("Inbox");
-
-            $numofPMs = get_number_of_unread_messages($user->username);
-            if ($numofPMs > 0) {
-                $inbox_text .= " " . sprintf(_("(%s unread)"), $numofPMs);
-            }
-            $links[] = ['text' => $inbox_text, 'url' => get_url_for_inbox()];
-        }
+        $links[] = ['text' => _("Stats"), 'url' => "stats/stats_central.php"];
 
-        if (get_url_for_forum()) {
-            $links[] = ['text' => _("Forums"), 'url' => get_url_for_forum()];
-        }
+        echo headerbar_text_array($links);
+        echo "  &#183; ";
 
-        if (!empty($blog_url)) {
-            $links[] = ['text' => _("Blog"), 'url' => $blog_url];
-        }
+        $account_links = [];
```

Comment:
Maybe just make this a single assign rather than appending? 
```
$account_links = [ ..., ... , ... ];
```

Reply:
Oh yeah, that's much better. And obvious in retrospect.